dmscripts/generate_framework_master_csv.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""Class used to output a framework master csv."""
+import csv
+from sys import stdout
+
+
+class GenerateMasterCSV(object):
+    """"Generate a master csv given a framework."""
+
+    static_fieldnames = ['supplier_id', 'supplier_dm_name', 'application_status', 'declaration_status']
+    lot_name_prefixes = ['draft', 'completed']
```

Comment:
Yeah it was like that in the script before. I watered it down a bit because I had trouble reading it but actually I'm happy to put it back in maybe with comments.